lib/plugins/Label_2P_POS.test.ts

```diff
@@ -0,0 +1,54 @@
+import { MessageDecoder } from '../MessageDecoder';
+import { Label_2P_POS } from './Label_2P_POS';
+
+describe('Label_2P Preamble POS', () => {
+
+  let plugin: Label_2P_POS;
+
+  beforeEach(() => {
+    const decoder = new MessageDecoder();
+    plugin = new Label_2P_POS(decoder);
+  });
+
+  test('variant 1', () => {
```

Comment:
this test will pass when https://github.com/airframesio/acars-decoder-typescript/pull/224 is merged